pkg/controllers/deprovisioning/controller.go

```diff
@@ -259,9 +259,9 @@ func (c *Controller) waitForReadiness(ctx context.Context, key nodeclaimutil.Key
 			// The NodeClaim got deleted after an initial eventual consistency delay
 			// This means that there was an ICE error or the Node initializationTTL expired
 			if errors.IsNotFound(err) && c.clock.Since(pollStart) > time.Second*5 {
-				return retry.Unrecoverable(fmt.Errorf("getting machine, %w", err))
+				return retry.Unrecoverable(fmt.Errorf("getting %s, %w", lo.Ternary(key.IsMachine, "machine", "nodeclaim"), err))
```

Comment:
I see we're still referring to "machine" here, is that intentional?

Reply:
Yes, we're referring to Machine here for nodes that are still represented as Machines. For those that are represented as NodeClaims, we use the "nodeclaim" string value